look in /usr/share/games/simutrans for game data

@DPATCH@

--- simutrans.orig/simmain.cc
+++ simutrans/simmain.cc
@@ -437,8 +437,9 @@
 		strcat( env_t::program_dir, PATH_SEPARATOR );
 	}
 	else {
-		strcpy( env_t::program_dir, argv[0] );
-		*(strrchr( env_t::program_dir, PATH_SEPARATOR[0] )+1) = 0;
+		// strcpy( env_t::program_dir, argv[0] );
+		// *(strrchr( env_t::program_dir, PATH_SEPARATOR[0] )+1) = 0;
+		strcpy(env_t::program_dir, "/usr/share/games/simutrans/");
 
 #ifdef __APPLE__
 		// change working directory from binary dir to bundle dir
